testpmd/hooks/install.yml

```diff
@@ -14,6 +28,7 @@
   community.kubernetes.k8s:
     state: present
     definition: "{{ lookup('template', 'templates/pdb.yml.j2') | from_yaml }}"
+    high_perf_runtime: "{{ runtime_class | default('') }}"
```

Comment:
Why are we passing high_perf_runtime here? In fact this task is failing in the last debug job because this variable is not expected here.